src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -212,6 +212,9 @@ public int OnQueryCloseSolution(object pUnkReserved, ref int pfCancel)
 
         public int OnBeforeCloseSolution(object pUnkReserved)
         {
+            // When closing solution (or closing VS), remove the temporary folder.
+            CleanupTemporaryFiles();
```

Comment:
>Cleanup [](start = 12, length = 7)

As a verb, "clean up" is two words, so: `CleanUpTemporaryFiles`. Or maybe just `RemoveTemporaryFiles`.